gridfs.go

```diff
@@ -657,7 +657,7 @@ func (file *GridFile) insertChunk(data []byte) {
 // an error, if any.
 func (file *GridFile) Seek(offset int64, whence int) (pos int64, err error) {
 	file.m.Lock()
-	debugf("GridFile %p: seeking for %s (whence=%d)", file, offset, whence)
```

Comment:
These are just cases of newer versions of the Go compiler no longer allowing broken format strings.